tools/summaries/places-short.json

```diff
@@ -0,0 +1,58 @@
+{
+  "city": {
```

Comment:
If you have the queries handy, can you document them in the README for future reference?

Reply:
I used the ranking API, but will document when we add other place types.